main_logic/core.py

```
@@ -560,9 +571,120 @@ async def handle_input_transcript(self, transcript: str):
                 self.message_cache_for_new_session.append({"role": self.master_name, "text": transcript.strip()})
             elif self.message_cache_for_new_session[-1]['role'] == self.master_name:
                 self.message_cache_for_new_session[-1]['text'] += transcript.strip()
+
+        # 在进入实际生成前，同步查询一次本轮即时负面偏好指令。
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)